### DISCUSSION TITLE
Tailwind CSS IntelliSense not working

### DISCUSSION THREAD

**Discussion by hujjumiyanji27** [tailwindlabs/tailwindcss#11100] [Help]
In my vs code I install the Tailwind CSS IntelliSense for autocomplete class for css but unfortunately its not working. Can you please help me how can i fix this error.

**Comment by bradlc** [comment]
Hey @hujjumiyanji27. Take a look at the [troubleshooting](https://github.com/tailwindlabs/tailwindcss-intellisense#troubleshooting) section of the extension readme. If you're still having issues after that then sharing your project details (and ideally the project itself as a GitHub repository) will make it much easier for others to help you out 👍
